xbmc/pvr/timers/PVRTimerInfoTag.h

```diff
@@ -233,7 +233,7 @@ namespace PVR
     std::string           m_strSummary;          /*!< @brief summary string with the time to show inside a GUI list */
     PVR_TIMER_STATE       m_state;               /*!< @brief the state of this timer */
     int                   m_iClientId;           /*!< @brief ID of the backend */
-    int                   m_iClientIndex;        /*!< @brief index number of the tag, given by the backend, -1 for new */
+    unsigned int          m_iClientIndex;        /*!< @brief index number of the tag, given by the backend, PVR_INVALID_TIMER_ID for new */
```

Comment:
the thing needs to be initialised with some value, or the compiler will complain. and instead of using a magic value, using a #define or const defined somewhere is the proper way to do it.

but the value should not be used for any checks in kodi, that's correct. it's something that's used by the add-on.